### DISCUSSION TITLE
Agent Starter error

### DISCUSSION THREAD

**Discussion by Rahul-0108** [iTwin/itwinjs-core#3119] [Q&A]
Hi , 
I am trying to run the Agent Starter: https://github.com/iTwin/agent-starter , and used the scope "itwinjs" as mentioned there
but getting this error:
![image](https://user-images.githubusercontent.com/53996840/151911884-0c8b8117-26f6-4617-a349-b5382203ddda.png)

**Comment by Rahul-0108** [comment]
Any  Solution @calebmshafer  ??

**Reply by Rahul-0108** [reply]
Any Soulution ??

**Reply by Rahul-0108** [reply]
Any Solution for this ??

**Reply by aruniverse** [reply]
Sorry for the delayed response. We're looking into this and will respond once we have a solution.

**Reply by calebmshafer** [reply]
Hey @Rahul-0108, can you increase the logging level that you're using by modifying this [line](https://github.com/iTwin/agent-starter/blob/master/src/main.ts#L11) to use `LogLevel.Trace` and then provide the output? This will give more information about each of the network requests being made which should help diagnose.

Also, could you please provide the iTwin.js version you're using?

**Reply by Rahul-0108** [reply]
The logs are Here @calebmshafer 
![image](https://user-images.githubusercontent.com/53996840/151911893-fba966ad-a572-452b-b52c-a89bd98c6500.png)

I am using the packages of version 2.19.27
And I have Added the Client Email of this ClientId to the iModel's Add Participants
